### PR TITLE
EWL-6782: Adding New Component for Blockquotes.

### PR DESCRIPTION
**Pull request by acidaniel**
## Ticket(s)

**Jira Ticket**
- [EWL-6782: Formatting blockquotes](https://issues.ama-assn.org/browse/EWL-6782)

## Description
Adding the Blockquote new component, styles and needed files to render properly the blockquote component

## To Test
- [ ] Checkout this branch and up the styleguide `gulp serve`
- [ ] Go to `http://localhost:3000/?p=atoms-block-quote`
- [ ] Ensure that the component is displayed correctly on all devices for mobile, tablet and desktop based on [Zeplin Designs](https://issues.ama-assn.org/browse/EWL-6782)

## Visual Regressions
N/A

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.

## Relevant Screenshots/GIFs
Selection_003

Selection_004

## Remaining Tasks
N/A

## Additional Notes
This is related to https://github.com/AmericanMedicalAssociation/ama-d8/pull/1623 we need both in order to test it

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/_patterns/01-atoms/block-quote/block-quote.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "text": "Many professional and amateur athletic activities, including contact sports, can put participants at risk of injury. Physicians can provide valuable information to help sports participants, dancers and others make informed decisions about whether to initiate or continue participating in such activities. any professional and amateur athletic activities, including contact sports, can put participants at risk of injury. Physicians can provide valuable information to help sports participants, dancers and others make informed decisions about whether to initiate or continue participating in such activities."
+}
```
Comment (russom-woldezghi) [inline styleguide/source/_patterns/01-atoms/block-quote/block-quote.json]: Add extra line break.

File: styleguide/source/_patterns/01-atoms/block-quote/block-quote.twig [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="blockquote">
+  <div class="blockquote__text">
+  {{ text }}
+  </div>
+</div>
```
Comment (russom-woldezghi) [inline styleguide/source/_patterns/01-atoms/block-quote/block-quote.twig]: Add extra line break.

File: styleguide/source/assets/scss/01-atoms/_block-quote.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    @include breakpoint($bp-small) {
+      max-width: 59%;
+    }
+  }
+
+  * {
+    margin-left: 25px;
+  }
+}
```
Comment (russom-woldezghi) [inline styleguide/source/assets/scss/01-atoms/_block-quote.scss]: Add extra line break.

File: styleguide/source/assets/scss/01-atoms/_block-quote.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  @include gutter($margin-bottom-half...);
+  @include gutter($margin-top-half...);
+
+  &:before {
+    content: '';
+    height: 100%;
+    background-color: $purple;
+    position: absolute;
+    width: 5px;
+
```
Comment (russom-woldezghi) [inline styleguide/source/assets/scss/01-atoms/_block-quote.scss]: Remove extra line.

File: styleguide/source/assets/scss/01-atoms/_block-quote.scss [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  &__text {
+    margin-left: 1.6%;
+
```
Comment (russom-woldezghi) [inline styleguide/source/assets/scss/01-atoms/_block-quote.scss]: Remove extra line.

File: styleguide/source/assets/scss/01-atoms/_block-quote.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    max-width: 100%;
+
```
Comment (russom-woldezghi) [inline styleguide/source/assets/scss/01-atoms/_block-quote.scss]: Remove extra line.

**Review by russom-woldezghi — DISMISSED** [review]
@acidaniel Looks good, just some formatting feedback.

**Review by joshuathorne — APPROVED** [review]
Russ' suggested changes made. Blockquote style displays for all breakpoints.

File: styleguide/source/_patterns/01-atoms/block-quote/block-quote.json (hunk 1 above)
Comment (joshuathorne) [reply styleguide/source/_patterns/01-atoms/block-quote/block-quote.json]: done

File: styleguide/source/assets/scss/01-atoms/_block-quote.scss (hunk 3 above)
Comment (joshuathorne) [reply styleguide/source/assets/scss/01-atoms/_block-quote.scss]: done

**Pull request merged**